how-to-docs/drone-how-to.md

```diff
@@ -315,39 +315,39 @@ Drone will automatically execute that step when a new pull request is raised.
 
 [Read more about Drone conditions](http://docs.drone.io/conditional-steps/).
 
-## Deploying to DSP
+## Deploying to ACP
 
-> Please note that this section assumes you have a separate repository containing your kube files as explained [here](https://github.com/UKHomeOffice/application-container-platform/blob/master/developer-docs/platform_introduction.md#define-a-deployment-for-your-application).
+> Please note that this section assumes that your kube files are in the same directory as your `.drone.yaml` file and your `deploy.sh` script.
 
-You can clone your kube repo as part of your pipeline with:
-
-```yaml
-predeploy_to_uat:
-  image: plugins/git
-  commands:
-    - git clone https://${GITHUB_TOKEN}:x-oauth-basic@github.com/UKHomeOffice/<your_repo>.git
-  when:
-    environment: uat
-    event: deployment
-```
-
-from now on, the repository is part of the workspace and is ready to be accessed by other steps in the pipeline.
-
-You can execute the deployment script in a new step with:
+You can execute the deployment script by adding a new step to your drone pipeline:
 
 ```yaml
 deploy_to_uat:
   image: quay.io/ukhomeofficedigital/kd:v0.2.3
-  environment:
-    - KUBE_NAMESPACE=<dev-induction>
+  secrets:
+    - kube_server
+    - kube_token
   commands:
-    - cd <kube-node-hello-world>
     - ./deploy.sh
   when:
     environment: uat
     event: deployment
 ```
 
+> If you do not already have a deployment script, you can create one by adding the following to a new file called `deploy.sh`:
```

Comment:
The deploy script will need to be created regardless. (So it might be good to put this at the beginning).

Reply:
It's also worth mentioning that the script below is just an example script, and that it'll need to be repurposed for your particular app/deployments/environments/etc.